features/users/user_dashboard.feature

```diff
@@ -65,7 +66,10 @@ Feature: User dashboard
     And I go to meatloaf's user page
   Then I should see "Newest Work"
     And I should not see "Oldest Work"
-    And I should see "Works (6)" within "#dashboard"
+    And I should see "Works (5)" within "#dashboard"
+  When I wait 11 seconds
```

Comment:
Waiting can be imprecise... if I set the expiration time too low, the tests might actually take longer, and the stale check (`see "Works (5)"`) will fail.

Ideally I'd like to use Timecop to control time precisely and instantly (`And it is currently 25 minutes from now`), but Timecop doesn't affect memcached because memcached is a separate process, but we can switch to [an in-memory, same-process cache store](https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-memorystore) which can time-travel with Timecop, except we cannot switch to the in-memory cache store because it uses [Ruby's built-in serialization that cannot handle IO objects](https://ruby-doc.org/core-2.6.5/Marshal.html#method-c-dump).

And we do cache IO objects: [paperclip attachments](https://github.com/thoughtbot/paperclip/issues/1114), on the [default Skin object](https://github.com/otwcode/otwarchive/blob/cccba97ecd7b4ab69ff22a00f3df9cddf7f22860/app/models/skin.rb#L504-L508).

So `Kernel::sleep` for now.